### DISCUSSION TITLE
打开chatgpt啥也没有，验证是否人类之后就没了。

### DISCUSSION THREAD

**Discussion by lkpopo** [tw93/Pake#332] [Q&A]
![image](https://user-images.githubusercontent.com/101188826/223056494-a5963df1-ca87-4de4-a677-b79b7ffcb66c.png)

**Comment by lkpopo** [comment]
已经解决了，右键刷新一下就好了

**Reply by marlonfan** [reply]
我这边也出现了, 用的是最新的包。。。

**Comment by marlonfan** [comment]
@tw93 Hi Bro, 这个是什么问题, 我这边最新的包也出现了... 没什么排查思路

**Reply by tw93** [reply]
重新关掉 然后打开 
或者右键刷新应该都可以

**Reply by marlonfan** [reply]
重试了多次确实不行…

**Reply by marlonfan** [reply]
mac是好的，win不行

**Reply by tw93** [reply]
辛苦帮我试试这个版本 https://github.com/tw93/Pake/releases/download/V1.0.5/ChatGPT_x64.msi

**Reply by marlonfan** [reply]
好的，我试试

**Reply by marlonfan** [reply]
> 辛苦帮我试试这个版本 https://github.com/tw93/Pake/releases/download/V1.0.5/ChatGPT_x64.msi

不行, Cloudflare的验证转了一个圈, 然后就变成logo无法点击

**Reply by MororoAoi** [reply]
我也是

**Comment by michael-kudrik** [comment]
This is also happening to me when I try to access a site that uses Cloudflare as verification. 
当我尝试访问使用 Cloudflare 作为验证的网站时，也会发生这种情况。

**Comment by mao-mao-yu** [comment]
> https://github.com/tw93/Pake/releases/download/V1.0.5/ChatGPT_x64.msi
这个链接我成功了
